app/javascript/react/containers/RestaurantFormContainer.js

```diff
@@ -0,0 +1,296 @@
+import React, { Component } from 'react';
+import CuisineTile from '../components/CuisineTile';
+import TextField from '../components/TextField'
+import TextArea from '../components/TextArea'
+import Dropzone from 'react-dropzone'
+
+class RestaurantFormContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      street: '',
+      city: '',
+      state: '',
+      zip: '',
+      phoneNumber: '',
+      email: '',
+      website: '',
+      file: [],
+      errors: {},
+      notices: {}
+
+    };
+
+    this.handleNameUpdate = this.handleNameUpdate.bind(this);
+    this.handleStreetUpdate = this.handleStreetUpdate.bind(this);
+    this.handleCityUpdate = this.handleCityUpdate.bind(this);
+    this.handleStateUpdate = this.handleStateUpdate.bind(this);
+    this.handleZipUpdate = this.handleZipUpdate.bind(this);
+    this.handlePhoneNumberUpdate = this.handlePhoneNumberUpdate.bind(this);
+    this.handleEmailUpdate = this.handleEmailUpdate.bind(this);
+    this.handleWebsiteUpdate = this.handleWebsiteUpdate.bind(this);
+    this.submitRestaurant = this.submitRestaurant.bind(this);
+    this.handleFormSubmit = this.handleFormSubmit.bind(this);
+    this.formIsValid = this.formIsValid.bind(this);
+    this.onDrop = this.onDrop.bind(this);
+    this.successfulFormSubmit = this.successfulFormSubmit.bind(this);
+  }
+
+  successfulFormSubmit(event) {
+    event.preventDefault()
+    this.setState({
+        name: '',
+        street: '',
+        city: '',
+        state: '',
+        zip: '',
+        phoneNumber: '',
+        email: '',
+        website: '',
+        file: [],
+        errors: {},
+        notices: {}
+
+      })
+
+  }
```

Comment:
Is this supposed to clear the form? If so, call it clear form, confusing name

Reply:
Sure, great suggestion, will rename the method

Reply:
Method renamed to clearForm()